Shubham_Dalal/HTML/ASSIGNMENT7/assignment7_login.css

```diff
@@ -0,0 +1,64 @@
+body {
+    padding: 0px;
+    margin: 0px;
+}
+h1 {
+    text-align: center;
+}
+header {
+    height: 50px;
+    width: auto;
+    background-color: rgba(31, 102, 98, 0.7);
+    display: flex;
+}
+header ul {
+    align-self: center;
+}
+header ul li {
+    list-style-type: none;
+    float: left;
+}
+header ul li a {
+    text-decoration: none;
+    color: white;
+    text-align: center;
+    padding: 20px;
+}
+li a:hover {
+    background-color: teal;
+}
+.center {
+    display: flex;
+    justify-content: center;
+}
+div img {
```

Comment:
if there are same classes then u can use the above style sheet why repeated classes in another file